DEV_GUIDE.md

```diff
@@ -184,10 +185,47 @@ Low level network and frame logging is turned off by default for better performa
 
 ## Building and pushing a Kroxylicious Container Image
```

Comment:
@k-wall Should I keep the heading same or something like -> `Building and pushing Kroxylicious Container Images`?

Reply:
Plural makes sense.